### PR TITLE
Adding demos directory where to store scripts for presenting KQueen

### PR DESCRIPTION
**Pull request by ruzickap**
Adding demos directory where to store scripts for presenting KQueen

### CODE REVIEW

File: demos/kqueen_azure-demo.sh [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+p  '#'
+p  '# Create a new Resource Group:'
+wait
+pe 'RESOURCE_GROUP_NAME="kqueen-demo-rg"'
+pe 'az group create --name "$RESOURCE_GROUP_NAME" --location westeurope'
+
+p  '#'
+p  '# Create a service principal that will store permissions to manage resources in the a specified subscription:'
+wait
```
Comment (naumvd95) [inline demos/kqueen_azure-demo.sh]: seems need to provide description to user, `time to set own creds`

File: demos/README.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This directory contains the scripts / commands / tools used for presenting the KQueen.
+
+Overview
+--------
+
+* `kqueen_azure_demo.sh` containes the commands used for configuring Azure, starting KQueen and deploying k8s to Azure via curl and API calls.
+
+
+Demos
```
Comment (naumvd95) [inline demos/README.rst]: CLI demos

Comment (naumvd95) [reply demos/README.rst]: coz we already have youtube videos

**Comment by naumvd95** [comment]
also, do user need to manually install https://asciinema.org/ ?if so, pls add instructions

File: demos/kqueen_azure-demo.sh (hunk 1 above)
Comment (ruzickap) [reply demos/kqueen_azure-demo.sh]: I do not understand what you mean... What should I fix exactly?

File: demos/kqueen_azure-demo.sh [hunk 3]
```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+
+################################################
+# include the magic
```
Comment (naumvd95) [inline demos/kqueen_azure-demo.sh]: pls describe main purpose of this tool for users to increase visibility

File: demos/kqueen_azure-demo.sh (hunk 1 above)
Comment (naumvd95) [reply demos/kqueen_azure-demo.sh]: how do user should define its credentials there for demo?

**Comment by ruzickap** [comment]
Asciinema is not required for the demo.
Asciinema was used to record the command line and produce the "screencast": https://asciinema.org/a/182231?autoplay=1

File: demos/kqueen_azure-demo.sh (hunk 1 above)
Comment (ruzickap) [reply demos/kqueen_azure-demo.sh]: Azure authentication is not part of the script. That's part of the "az" command and it's settings. 
This is more KQueen demo than the Azure demo. 

The login/password for the service principal are defined here:

```
pe 'SECRET="my_secret_password"'
pe 'SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"'
```

Comment (naumvd95) [reply demos/kqueen_azure-demo.sh]: okok, if so, maybe need to add smth like:
> +p  '# Define your Azure credentials here?'

Comment (ruzickap) [reply demos/kqueen_azure-demo.sh]: I replaced `SECRET` variable by `SERVICE_PRINCIPAL_PASSWORD` to have it more clear:

```
pe 'SERVICE_PRINCIPAL_PASSWORD="my_secret_password"'
pe 'SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"'
pe 'az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "$SERVICE_PRINCIPAL_NAME" --password "$SERVICE_PRINCIPAL_PASSWORD"'
```

File: demos/kqueen_azure-demo.sh [hunk 4]
```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+################################################
+# include the demo-magic script
+################################################
+# demo-magic.sh is a handy shell script that enables you to script repeatable demos in a bash environment so you don't have to type as you present
```
Comment (ekhomyakova) [inline demos/kqueen_azure-demo.sh]: I wouldn't understand this description if I haven't seen how it works. I think smth like that is more clear:
`..in a bash environment. It simulates typing of your commands, so you don't have to type them by yourself when you are presenting`

File: demos/kqueen_azure-demo.sh [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+# Rather than trying to type commands when presenting you simply script them and let demo-magic.sh run them for you
+test -f ./demo-magic.sh || curl --silent https://raw.githubusercontent.com/paxtonhare/demo-magic/master/demo-magic.sh > demo-magic.sh
+. ./demo-magic.sh -n
+
+################################################
+# Configure the options
+################################################
+
+#
+# speed at which to simulate typing. bigger num = faster
```
Comment (ekhomyakova) [inline demos/kqueen_azure-demo.sh]: Do you have a more specific description? E.g. it's symbols per minute or smth like that

File: demos/kqueen_azure-demo.sh [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+#
+TYPE_SPEED=40
+
+# Uncomment to run non-interactively
```
Comment (ekhomyakova) [inline demos/kqueen_azure-demo.sh]: It's uncommented already

File: demos/kqueen_azure-demo.sh [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+export PROMPT_TIMEOUT=0
+
+# No wait
```
Comment (ekhomyakova) [inline demos/kqueen_azure-demo.sh]: No wait for what?

File: demos/README.rst [hunk 8]
```diff
@@ -0,0 +1,19 @@
+KQueen Demos
+============
+
+This directory contains the scripts / commands / tools used for presenting the KQueen.
+
+Overview
+--------
+
+* `kqueen_azure_demo.sh` containes the commands used for configuring Azure, starting KQueen and deploying k8s to Azure via curl and API calls.
```
[the remark this answers is not in the document]
Comment (ruzickap) [reply demos/README.rst]: Fixed... Thank you.

**Comment by katyafervent** [comment]
Thanks for the commit Could you please rebase your change?

**Comment by ruzickap** [comment]
Rebase done... :-)

**Pull request merged**